plugin.yaml

```diff
@@ -246,10 +243,14 @@ node_types:
           implementation: helm.cloudify_helm.tasks.remove_repo
           inputs:
             flags:
-              description: >
-                Flags to add only to `helm uninstall` command.The format is the same as "flags".
-              default: []
-
+              <<: *flags_input
+      helm:
```

Comment:
I wondered where to put this because `helm repo update` updates all repos for the client. 
but i think under Repo type its the right place.

Reply:
Is it on the same indentation as `cloudify.interfaces.lifecycle` then it is fine.